src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsCommentsListener.php

```diff
@@ -0,0 +1,126 @@
+	 *
+	 * @param   Event  $event  Event object
+	 *
+	 * @return  void
+	 *
+	 * @since   1.0
+	 */
+	public function onCommentAfterCreate(Event $event)
+	{
+		// Pull the arguments array
+		$arguments = $event->getArguments();
+
+		// Add a RTC label if the item is in that status
+		$this->checkRTClabel($arguments['hookData'], $arguments['github'], $arguments['logger'], $arguments['project'], $arguments['table']);
+	}
+
+	/**
+	 * Event for after Comments requests are updated in the application
+	 *
+	 * @param   Event  $event  Event object
+	 *
+	 * @return  void
+	 *
+	 * @since   1.0
+	 */
+	public function onCommentAfterUpdate(Event $event)
+	{
+		// Pull the arguments array
+		$arguments = $event->getArguments();
+
+		$this->checkNoCodelabel($arguments['hookData'], $arguments['github'], $arguments['logger'], $arguments['project'], $arguments['table']);
+
+		// Add a RTC label if the item is in that status
+		$this->checkRTClabel($arguments['hookData'], $arguments['github'], $arguments['logger'], $arguments['project'], $arguments['table']);
+	}
+
+	/**
+	 * Adds a "No Code Attached Yet" label
+	 *
+	 * @param   object       $hookData  Hook data payload
+	 * @param   Github       $github    Github object
+	 * @param   Logger       $logger    Logger object
+	 * @param   object       $project   Object containing project data
+	 * @param   IssuesTable  $table     Table object
+	 *
+	 * @return  void
+	 *
+	 * @since   1.0
+	 */
+	protected function checkNoCodelabel($hookData, Github $github, Logger $logger, $project, IssuesTable $table)
+	{
+		// Set some data
+		$label      = 'No Code Attached Yet';
+		$labels     = array();
+		$labelIsSet = $this->checkLabel($hookData, Github $github, Logger $logger, $project, IssuesTable $table, $label);
+
+		if ($labelIsSet == false)
+		{
+			// Add the RTC label as it isn't already set
+			$labels[] = $label;
+			$this->addLabels($hookData, Github $github, Logger $logger, $project, IssuesTable $table, $labels);
+		}
+	}
+
+	/**
+	 * Checks for the RTC label
+	 *
+	 * @param   object       $hookData  Hook data payload
+	 * @param   Github       $github    Github object
+	 * @param   Logger       $logger    Logger object
+	 * @param   object       $project   Object containing project data
+	 * @param   IssuesTable  $table     Table object
+	 *
+	 * @return  void
+	 *
+	 * @since   1.0
+	 */
+	protected function checkRTClabel($hookData, Github $github, Logger $logger, $project, IssuesTable $table)
+	{
+		// Set some data
+		$label      = 'RTC';
+		$labels     = array();
+		$labelIsSet = $this->checkLabel($hookData, Github $github, Logger $logger, $project, IssuesTable $table, $label);
+
+		// Validation, if the status isn't RTC or the Label is set then go no further
+		if ($labelIsSet == true && $table->status != 4)
+		{
+			// Remove the RTC label as it isn't longer set to RTC
+			$labels[] = $label;
+			$this->removeLabel($hookData, Github $github, Logger $logger, $project, IssuesTable $table, $labels);
```

Comment:
You should remove type hinting here. And also do not forget to remove `$table` as it is not used.